### PR TITLE
Accept `--terminal-total-difficuty-override` as decimal

### PR DESCRIPTION
**Pull request by paulhauner**
## Issue Addressed

NA

## Proposed Changes

As per the presently un-approved https://github.com/ethereum/consensus-specs/pull/2645, parse `--terminal-total-difficuty-override` as a decimal integer instead of 0x-prefixed hex.

Although we don't have clear guidance on this from the spec, I understand that this is the most convenient format for the merge event.

## Additional Info

NA

### CODE REVIEW

**Review by zedt3ster — APPROVED** [review]

**Comment by zedt3ster** [comment]
Please do not merge for now.

**Comment by paulhauner** [comment]
> Please do not merge for now.

@zedt3ster asked to delay merging this since it's a breaking change and would have been really annoying during the F2F.

The F2F is over now, and I'm keen to get this merged before we get an influx of new users. It'll only get harder to change from this point. I've run this past Proto, who is involved in spinning up the new testnets. He has given me the green light.

All of this considered, I'm going to merge this now.

**Pull request merged**
